aodncore/pipeline/steps/resolve.py

```diff
@@ -193,18 +206,25 @@ class MapManifestResolveRunner(BaseManifestResolveRunner):
         /path/to/source/file2,destination/path/for/upload2
 
     """
-    PATH_SPLIT_CHAR = ','
+    schema = tableschema.Schema({'fields': [
+        {
+            'name': 'local_path',
+            'type': 'string',
+            'constraints': {'required': 'true'}
+        },
+        {
+            'name': 'dest_path',
+            'type': 'string',
+            'constraints': {'required': 'true'}
+        }
+    ]})
 
     def run(self):
-        with open(self.input_file, 'r') as f:
-            for line_newline in f:
-                line = line_newline.rstrip(os.linesep)
-                src, dest_path = line.split(self.PATH_SPLIT_CHAR, 1)
-
-                abs_path = self.get_abs_path(src)
-                fileobj = PipelineFile(abs_path, dest_path=dest_path)
-
-                self._collection.add(fileobj)
+        table = tableschema.Table(self.input_file, headers=self.schema.field_names, schema=self.schema, format='csv')
+        for local_path, dest_path in table.iter(exc_handler=self._exc_handler):
```

Comment:
Don't we need to handle schema errors here too, as you've done in `DeleteManifestResolveRunner`? 
e.g. if either the local_path or dest_path are empty?
Or are these cases adequately handled by `PipelineFile.__init__`?

Reply:
Yeah good point, I think an empty string would end up with an error which wouldn't be handled.

Reply:
@mhidas I've refactored the validation logic into a base class which I think should make it flexible enough to handle pretty much any kind of CSV we want to handle. The child classes define the schema of the CSV, and a "row handler" method which translates each row into a PipelineFile.